config.example.toml

```diff
@@ -245,6 +245,11 @@ type = "Loss"
 # Default: []
 pattern = [0.1, 0.2]
 
+# Delay Per-Packet Cell Example
+[cells.down_3]
+type = "DelayPerPacket"
+
+FromFile = "./assets/delay-per-packet.toml"
```

Comment:
Please use the name `trace` instead of `FromFile` as to be compatible with other cells. Also we prefer snake_case in config file.